.devcontainer/devcontainer.json

```
@@ -1,79 +1,43 @@
-// For format details, see https://aka.ms/devcontainer.json. For config options,
-// see documentation located at the following URL:
-// https://github.com/OpenINF/openinf-docker-fish/tree/main/library-scripts/docs
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-outside-of-docker
 {
-  "name": "Jekyll (OpenINF Community)",
-  "build": {
-    "dockerfile": "Dockerfile"
-  },
-  "runArgs": ["--cap-add=SYS_ADMIN", "--security-opt", "seccomp=unconfined"],
-  // Configure tool-specific properties.
-  "customizations": {
-    // Configure properties specific to VS Code.
-    "vscode": {
-      // Set *default* container-specific settings.json values on container
-      // create.
-      "settings": {
-        "terminal.integrated.profiles.linux": {
-          "fish": {
-            "path": "fish"
-          }
-        },
-        "terminal.integrated.defaultProfile.linux": "fish"
-      },
+	"name": "Docker outside of Docker",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
```

```suggestion
	"image": "mcr.microsoft.com/devcontainers/base:bullseye-slim",
```

There has already been good reason why we did not use these prebuilt;
we will have to re-link to our own as we were doing previously:

- https://ghithub.com/OpenINF/docker-fisher

Will probably be re-based on 0-vuln slim hopefully&hellip;

Refs: https://twitter.com/DerekNonGeneric/status/1739670519174054005

---

i now see why the chainguard solution does not appeal to us;
or more-specifically those in a certification environment

> There has already been good reason why we did not use these prebuilt;
> we will have to re-link to our own as we were doing previously:

before, it was because it simply was not open source nor what we
specifically needed ahaha